app/assets/scss/_updates.scss

```diff
@@ -1,7 +1,3 @@
-#input-clarification_question {
-  height: 20.75em;
-}
```

Comment:
The class is built in the brief-responses frontend through a combination of https://github.com/alphagov/digitalmarketplace-brief-responses-frontend/blob/master/app/main/views/briefs.py#L73 which passes through to https://github.com/alphagov/digitalmarketplace-frontend-toolkit/blob/master/toolkit/templates/forms/textbox.html#L64. The same thing doesn't appear to exist any more in supplier-frontend (presumably because the functionality was moved to brief-responses).

Reply:
We may be using it for when a buyer adds a brief clarification question which is likely in this application (they get sent them by email from the supplier form and then fill in what the question is and what the answer is to post it). 

Regardless, it's unlikely this class is being used.

Reply:
We're using it here https://github.com/alphagov/digitalmarketplace-supplier-frontend/blob/master/app/templates/frameworks/updates.html#L133 where suppliers can ask clarification questions about a framework.

Reply:
Ah cool, so we do have an area that it seems appropriate for but technically we aren't using the class due to a underscore mismatch (for which we can't tell if is intentional or unintentional)?

Reply:
Huh, no, I'm saying that **_we are_** using this ID in the supplier app.

<img width="826" alt="screen shot 2017-12-12 at 12 10 11" src="https://user-images.githubusercontent.com/2920760/33883884-773f7f94-df35-11e7-9612-b7d90c1c696f.png">

Reply:
Ah OK, thanks I understand now.